source/CIGameScene.cpp

```diff
@@ -99,7 +110,7 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     _massHUD  = std::dynamic_pointer_cast<scene2::Label>(_assets->get<scene2::SceneNode>("game_hud"));
 
     // Create the planet model
-    _planet = PlanetModel::alloc(dimen.width/2, dimen.height/2, CIColor::getNoneColor(), 3);
+    _planet = PlanetModel::alloc(dimen.width / 2, dimen.height / 2, CIColor::getNoneColor(), MAX_PLANET_LAYERS, gravStrength, winCondition);
```

Comment:
Could you change the name winCondition to something a bit more specific. Maybe like winningPlanetMass or planetMassToWin

Reply:
I wasn't sure if we were planning to switch to using number of layers instead of the planet's mass to determine winning. Ill update the code to go along with using the planet mass